app/src/modules/settings/routes/flows/components/logs-sidebar-detail.vue

```diff
@@ -186,7 +186,7 @@ function onToggle(open: boolean) {
 							<pre class="json selectable">{{ step.options }}</pre>
 						</v-detail>
 
-						<v-detail v-if="step.data" :label="t('payload')">
+						<v-detail v-if="step.data !== null" :label="t('payload')">
```

Comment:
I wonder if it should be `undefined` instead as `null` could be a returned value 🤔 

```suggestion
						<v-detail v-if="step.data !== undefined" :label="t('payload')">
```

Reply:
We default to `null` for any undefined value above, see https://github.com/directus/directus/blob/eba1939ab4a15b3a06851a3767a3c3ec6f2e62e0/app/src/modules/settings/routes/flows/components/logs-sidebar-detail.vue#L95

Reply:
Based on [L95](https://github.com/directus/directus/pull/24231/files#diff-353425b538fe82f49352f0b1fd681a2bca344da6355a529736fedc093d41343dR95) it can only ever be null or a value